package/MDAnalysis/analysis/rms.py

```diff
@@ -101,7 +101,7 @@
    ax.plot(time, rmsd[5], 'b--', label="NMP")
    ax.legend(loc="best")
    ax.set_xlabel("time (ps)")
-   ax.set_ylabel(r"RMSD ($\AA$)")
+   ax.set_ylabel(r"RMSD ($\\\\AA$)")
```

Comment:
?

Reply:
Only one backslash throws a warning, so two are needed. 4 is definitely two much ;).